core/tauri-macros/src/command/wrapper.rs

```diff
@@ -210,14 +240,22 @@ pub fn wrapper(attributes: TokenStream, item: TokenStream) -> TokenStream {
 ///
 /// [`tauri::command`]: https://docs.rs/tauri/*/tauri/runtime/index.html
 fn body_async(function: &ItemFn, invoke: &Invoke, case: ArgumentCase) -> syn::Result<TokenStream2> {
-  let Invoke { message, resolver } = invoke;
-  parse_args(function, message, case).map(|args| {
+  let Invoke {
+    id,
+    message,
+    resolver,
+  } = invoke;
+  parse_args(id, function, message, case).map(|args| {
     quote! {
+      use tracing::Instrument;
+
+      let span = tracing::debug_span!("ipc::request::run", id = #id.0);
```

Comment:
```suggestion
      let span = tracing::debug_span!("ipc::request::handler", id = #id.0);
```

I would prefer we call this `ipc::request::handler`, to me its more descriptive than `run`

Reply:
there's already a ipc::request::handler somewhere else in the code :D

Reply:
ah yeah just noticed this too, how about `inner` then? idk

Reply:
Maybe call it `handler::run`?
handler means our Tauri IPC handler is running, and `run` means the actual command is running.